packages/camel-catalog/src/main/java/io/kaoto/camelcatalog/generator/CatalogGenerator.java

```diff
@@ -74,14 +74,17 @@ public CatalogDefinition generate() {
         processKameletsCRDs(catalogDefinition);
 
         try {
-            String filename = String.format("%s-%s.json", "index",
-                    Util.generateHash(catalogDefinition.toString()));
+            String content = jsonMapper.writeValueAsString(catalogDefinition);
```

Comment:
I acknowledge it was originally like this, but shouldn't we move this line right before where we're generating the hash?

```java
            String content = jsonMapper.writeValueAsString(catalogDefinition);

            String filename = String.format("%s-%s.json", "index",
                    Util.generateHash(content));
```

Reply:
done!